src/BloomExe/Publish/PublishView.cs

```diff
@@ -526,9 +533,15 @@ private void OnPrint_Click(object sender, EventArgs e)
 				{
 					using (var dlg = new SamplePrintNotification())
 					{
+#if __MonoCS__
+						_pdfViewer.PrintFinished -= FormActivatedAfterPrintDialog;
```

Comment:
I doubt we need to remove the event handler on Mono. The reason for doing it on windows is that the main window activates when the dialog closes. But the print process won't raise finished at that point.